plugins/services/src/js/reducers/serviceForm/JSONReducers/__tests__/UnknownVolumes-test.js

```diff
@@ -0,0 +1,137 @@
+const { ADD_ITEM } = require("#SRC/js/constants/TransactionTypes");
+const UnknownVolumes = require("../UnknownVolumes");
+
+describe("UnknownVolumes", function() {
+  describe("#JSONParser", function() {
```

Comment:
Do we need all `describe` in here? Seems like adding more complexity than clearly.

When trying to read and understand the units tests I'm getting confused would be clearer if the `it` description contained the info needed to understand what needs to be tested.

```
describe("UnknownVolumes", function() {
  describe("#JSONParser", function() {
    it("returns an empty array with empty input", function() {
      expect(UnknownVolumes.JSONParser({})).toEqual([]);
    });
  });
});
```